system-parachains/encointer/src/lib.rs

```diff
@@ -1023,3 +1008,237 @@ fn test_ed_is_one_tenth_of_relay() {
 	let encointer_ed = ExistentialDeposit::get();
 	assert_eq!(relay_ed / 10, encointer_ed);
 }
+
+#[test]
+fn test_constants_compatiblity() {
+	assert_eq!(
+		::kusama_runtime_constants::currency::EXISTENTIAL_DEPOSIT,
+		system_parachains_constants::kusama_runtime_constants::currency::EXISTENTIAL_DEPOSIT
+	);
+	assert_eq!(
+		::kusama_runtime_constants::currency::deposit(5, 3),
+		system_parachains_constants::kusama_runtime_constants::currency::deposit(5, 3)
+	);
+	assert_eq!(
+		::system_parachains_constants::AVERAGE_ON_INITIALIZE_RATIO * 1u32,
+		system_parachains_constants::AVERAGE_ON_INITIALIZE_RATIO * 1u32
+	);
+	assert_eq!(
+		::system_parachains_constants::NORMAL_DISPATCH_RATIO * 1u32,
+		system_parachains_constants::NORMAL_DISPATCH_RATIO * 1u32
+	);
+	assert_eq!(
+		::system_parachains_constants::MAXIMUM_BLOCK_WEIGHT.encode(),
+		system_parachains_constants::MAXIMUM_BLOCK_WEIGHT.encode()
+	);
+	assert_eq!(::system_parachains_constants::MINUTES, system_parachains_constants::MINUTES);
+	assert_eq!(::system_parachains_constants::HOURS, system_parachains_constants::HOURS);
+	assert_eq!(::system_parachains_constants::DAYS, system_parachains_constants::DAYS);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::SYSTEM_PARA_EXISTENTIAL_DEPOSIT,
+		system_parachains_constants::kusama::currency::SYSTEM_PARA_EXISTENTIAL_DEPOSIT
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::UNITS,
+		system_parachains_constants::kusama::currency::UNITS
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::QUID,
+		system_parachains_constants::kusama::currency::QUID
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::CENTS,
+		system_parachains_constants::kusama::currency::CENTS
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::MILLICENTS,
+		system_parachains_constants::kusama::currency::MILLICENTS
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::currency::system_para_deposit(5, 3),
+		system_parachains_constants::kusama::currency::system_para_deposit(5, 3)
+	);
+	assert_eq!(
+		::system_parachains_constants::kusama::fee::calculate_weight_to_fee(
+			&::system_parachains_constants::MAXIMUM_BLOCK_WEIGHT
+		),
+		system_parachains_constants::kusama::fee::calculate_weight_to_fee(
+			&system_parachains_constants::MAXIMUM_BLOCK_WEIGHT
+		)
+	);
+}
+
+// TODO: Encointer pallets does not have compatible `polkadot-sdk` versions,
```

Comment:
it can be very easy to forget to update this when something updated in constants. may be we should try to drop some dependancies from constant crates. Should we have at least an issue for thus?